### PR TITLE
adding cupy.searchsorted 

### PR DESCRIPTION
**Pull request by Onkar627**

### CODE REVIEW

**Comment by emcastillo** [comment]
`cupy.searchsorted` is already implemented

https://github.com/cupy/cupy/blob/master/cupy/_sorting/search.py#L342-L371

File: cupy/_io/formatting.py [hunk 1]
```diff
@@ -65,6 +65,21 @@ def format_float_positional(x, *args, **kwargs):
...
+    corresponding elements in `v` were inserted before the indices, the
+    order of `a` would be preserved.
+    Assuming that `a` is sorted:
+   
+    `side`  returned index `i` satisfies
+
+    left    ``a[i-1] < v <= a[i]``
+    right   ``a[i-1] <= v < a[i]``
+    ======  ============================
+    return cupy.searchsorted(a, 'searchsorted', v, side=side, sorter=sorter)
```
Comment (emcastillo) [inline cupy/_io/formatting.py:81]:
```suggestion
    """
    return cupy.searchsorted(a, 'searchsorted', v, side=side, sorter=sorter)
```

**Comment by emcastillo** [comment]
This needs to be in `cupy.ndarray` which is in the `cupy._core/core.pyx` file

**Comment by Onkar627** [comment]
> This needs to be in `cupy.ndarray` which is in the `cupy/_core/core.pyx` file

**Pull request closed without merging**